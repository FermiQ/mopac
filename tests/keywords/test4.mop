1SCF
comment1
comment2
He  0.0 0.0 0.0

1SCF OPEN(2,1)
comment1
comment2
He  0.0 0.0 0.0
