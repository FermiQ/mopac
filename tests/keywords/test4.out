 *******************************************************************************
 ** Site#: 26757        For non-commercial use only    Version 21.051M 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 21.051M, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left: 207**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Wed Jul 28 17:40:10 2021  *
 *  1SCF       - DO 1 SCF AND THEN STOP 
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY 7200.0 SECONDS REQUESTED
 *******************************************************************************
1SCF
 comment1
 comment2
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1      He          0.00000000  *   0.00000000  *   0.00000000  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1        He          0.0000      0.0000      0.0000

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 https://link.springer.com/article/10.1007/s00894-012-1667-x

           Empirical Formula: He  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 1


 -------------------------------------------------------------------------------
 1SCF
 comment1
 comment2


     1SCF WAS SPECIFIED, SO BFGS WAS NOT USED                 
     SCF FIELD WAS ACHIEVED                                   


                              PM7 CALCULATION
                                                       MOPAC2016 (Version: 21.051M)
                                                       Wed Jul 28 17:40:10 2021
                                                       No. of days remaining = 207




          FINAL HEAT OF FORMATION =          0.00000 KCAL/MOL =       0.00000 KJ/MOL


          TOTAL ENERGY            =        -54.09664 EV
          ELECTRONIC ENERGY       =        -54.09664 EV  POINT GROUP:     R3  
          CORE-CORE REPULSION     =          0.00000 EV
          COSMO AREA              =         33.80 SQUARE ANGSTROMS
          COSMO VOLUME            =         18.48 CUBIC ANGSTROMS

          IONIZATION POTENTIAL    =         22.325670 EV
          HOMO LUMO ENERGIES (EV) =        -22.326 16.247
          NO. OF FILLED LEVELS    =          1
          MOLECULAR WEIGHT        =          4.0026


          SCF CALCULATIONS        =          1
 
          WALL-CLOCK TIME         =          0.023 SECONDS
          COMPUTATION TIME        =          0.009 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1      He          0.00000000  *   0.00000000  *   0.00000000  *

                             CARTESIAN COORDINATES

   1   He        0.000000000     0.000000000     0.000000000


           Empirical Formula: He  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  


                  EIGENVALUES  
 -22.32567  16.24650  16.24650  16.24650


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1         He           0.000000        2.0000     2.00000     0.00000
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.000     0.000     0.000     0.000
 HYBRID         0.000     0.000     0.000     0.000
 SUM            0.000     0.000     0.000     0.000


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1 He   2.00000   0.00000   0.00000   0.00000
 *******************************************************************************
 ** Site#: 26757        For non-commercial use only    Version 21.051M 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 21.051M, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left: 207**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Wed Jul 28 17:40:10 2021  *
 *  OPEN(N,N)- THERE ARE 2 ELECTRONS IN 1 LEVELS
 *  1SCF       - DO 1 SCF AND THEN STOP 
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY 7200.0 SECONDS REQUESTED
 *******************************************************************************
1SCF OPEN(2,1)
 comment1
 comment2
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1      He          0.00000000  *   0.00000000  *   0.00000000  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1        He          0.0000      0.0000      0.0000

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 https://link.springer.com/article/10.1007/s00894-012-1667-x

           Empirical Formula: He  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  

      THERE ARE    0 DOUBLY FILLED LEVELS


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 0


                       NO. OF LEVELS WITH OCCUPANCY 2.000  =  1

 -------------------------------------------------------------------------------
 1SCF OPEN(2,1)
 comment1
 comment2


     1SCF WAS SPECIFIED, SO BFGS WAS NOT USED                 
     SCF FIELD WAS ACHIEVED                                   


                              PM7 CALCULATION
                                                       MOPAC2016 (Version: 21.051M)
                                                       Wed Jul 28 17:40:10 2021
                                                       No. of days remaining = 207




          FINAL HEAT OF FORMATION =       1247.49711 KCAL/MOL =    5219.52792 KJ/MOL


          TOTAL ENERGY            =          0.00000 EV
          ELECTRONIC ENERGY       =          0.00000 EV  POINT GROUP:     R3  
          CORE-CORE REPULSION     =          0.00000 EV
          COSMO AREA              =         33.80 SQUARE ANGSTROMS
          COSMO VOLUME            =         18.48 CUBIC ANGSTROMS

          NO. OF FILLED LEVELS    =          1
          MOLECULAR WEIGHT        =          4.0026


          SCF CALCULATIONS        =          1
 
          WALL-CLOCK TIME         =          0.031 SECONDS
          COMPUTATION TIME        =          0.016 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1      He          0.00000000  *   0.00000000  *   0.00000000  *

                             CARTESIAN COORDINATES

   1   He        0.000000000     0.000000000     0.000000000


           Empirical Formula: He  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  


                  EIGENVALUES  
 -31.77097  -5.85638  -5.85638  -5.85638


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1         He           2.000000        0.0000     0.00000     0.00000
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.000     0.000     0.000     0.000
 HYBRID         0.000     0.000     0.000     0.000
 SUM            0.000     0.000     0.000     0.000


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1 He   0.00000   0.00000   0.00000   0.00000

 **********************
 *                    *
 * JOB ENDED NORMALLY *
 *                    *
 **********************



 TOTAL JOB TIME:             0.06 SECONDS

 == MOPAC DONE ==
